docs/cloud/serverless.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Serverless 
+layout: default
+parent: Cloud
+has_children: false
+nav_order: 14 
+---
+
+# Serverless
+*Currently in preview*
```

Comment:
Use the callout shortcode rather than italics or whatnot (and tere's an existing suggestion to do something like below in my earlier review)

```
{: .note}
Serverless architecture is currently in preview and should not be used with production data.
```